packages/vuetify/src/directives/touch/index.ts

```diff
@@ -1,5 +1,10 @@
-import { VNodeDirective, VNode } from 'vue/types/vnode'
+// Libraries
+import { DirectiveBinding, ObjectDirective } from 'vue'
+
+// Utilities
 import { keys } from '../../util/helpers'
+
+// Types
 import { TouchHandlers, TouchValue, TouchWrapper } from 'types'
```

Comment:
Since you are upgrading this file, it should follow the new guidelines for imports. New lines, alpha ordered, etc.

Reply:
every import importing 2 or more things should have newlines?